plugins/com.contrastsecurity.ide.eclipse.ui/src/com/contrastsecurity/ide/eclipse/ui/internal/views/FilterDialog.java

```diff
@@ -0,0 +1,714 @@
+				dateTimeFrom.setHours(localDateTimeMinusHour.getHour());
+				dateTimeFrom.setMinutes(localDateTimeMinusHour.getMinute());
+				dateTimeFrom.setSeconds(localDateTimeMinusHour.getSecond());
+
+				break;
+			case Constants.LAST_DETECTED_DAY:
+				LocalDateTime localDateTimeMinusDay = localDateTime.minusDays(1);
+
+				dateTimeFrom.setYear(localDateTimeMinusDay.getYear());
+				dateTimeFrom.setDay(localDateTimeMinusDay.getDayOfMonth());
+				dateTimeFrom.setMonth(localDateTimeMinusDay.getMonthValue() - 1);
+				dateTimeFrom.setHours(localDateTimeMinusDay.getHour());
+				dateTimeFrom.setMinutes(localDateTimeMinusDay.getMinute());
+				dateTimeFrom.setSeconds(localDateTimeMinusDay.getSecond());
+				break;
+			case Constants.LAST_DETECTED_WEEK:
+				LocalDateTime localDateTimeMinusWeek = localDateTime.minusWeeks(1);
+
+				dateTimeFrom.setYear(localDateTimeMinusWeek.getYear());
+				dateTimeFrom.setDay(localDateTimeMinusWeek.getDayOfMonth());
+				dateTimeFrom.setMonth(localDateTimeMinusWeek.getMonthValue() - 1);
+				dateTimeFrom.setHours(localDateTimeMinusWeek.getHour());
+				dateTimeFrom.setMinutes(localDateTimeMinusWeek.getMinute());
+				dateTimeFrom.setSeconds(localDateTimeMinusWeek.getSecond());
+				break;
+			case Constants.LAST_DETECTED_MONTH:
+				LocalDateTime localDateTimeMinusMonth = localDateTime.minusMonths(1);
+
+				dateTimeFrom.setYear(localDateTimeMinusMonth.getYear());
+				dateTimeFrom.setDay(localDateTimeMinusMonth.getDayOfMonth());
+				dateTimeFrom.setMonth(localDateTimeMinusMonth.getMonthValue() - 1);
+				dateTimeFrom.setHours(localDateTimeMinusMonth.getHour());
+				dateTimeFrom.setMinutes(localDateTimeMinusMonth.getMinute());
+				dateTimeFrom.setSeconds(localDateTimeMinusMonth.getSecond());
+				break;
+			case Constants.LAST_DETECTED_YEAR:
+				LocalDateTime localDateTimeMinusYear = localDateTime.minusYears(1);
+
+				dateTimeFrom.setYear(localDateTimeMinusYear.getYear());
+				dateTimeFrom.setDay(localDateTimeMinusYear.getDayOfMonth());
+				dateTimeFrom.setMonth(localDateTimeMinusYear.getMonthValue() - 1);
+				dateTimeFrom.setHours(localDateTimeMinusYear.getHour());
+				dateTimeFrom.setMinutes(localDateTimeMinusYear.getMinute());
+				dateTimeFrom.setSeconds(localDateTimeMinusYear.getSecond());
+				break;
+			case Constants.LAST_DETECTED_CUSTOM:
+				dateTimeFrom.setEnabled(true);
+				dateTimeTo.setEnabled(true);
+				break;
+			}
+		}
+	};
+
+	public FilterDialog(Shell parentShell, ContrastSDK contrastSDK, Servers servers, Applications applications) {
+		super(parentShell);
+
+		extendedContrastSDK = (ExtendedContrastSDK) contrastSDK;
+		this.servers = servers;
+		this.applications = applications;
+	}
+
+	private String getOrgUuid() {
+		String orgUuid = null;
+		try {
+			orgUuid = ContrastCoreActivator.getSelectedOrganizationUuid();
+		} catch (Exception e) {
+			ContrastUIActivator.log(e);
+		}
+		return orgUuid;
+	}
+
+	private void createApplicationCombo(Composite composite, String orgUuid) {
```

Comment:
Remove **orgUuid** from function definition if its not going to be used.

Reply:
Done.